src/main/java/medicaldoctor/core/AppSession.java

```diff
@@ -34,18 +34,71 @@
     private static Patient currentPatient;
     private static PatientVisit currentVisit;
     private static NewUserResult newUserResult;
-
+    private static User adminUserSelection;
```

Comment:
a few points
1.) i already have a "currentPatient" thing, you should always  look  around at what exists in the code.
i do prefer your naming though. but then it should replace what i had, not leave dead code around.
2.) the flags are unnecessary, as you can just check the selection object.
3.) why so many user selections? we can't really look at more than one at a time, right?



throughout all these changes, i know we are trying to rush to get this done, but that doesn't mean create the messiest code possible.

Reply:
The flags are necessary because of theres so many layers searching you can do (within TableViews). Actions like this don't deal with ActiveUser. So you have to set a flag to load different things. To prevent collisions between them you can set flags to differentiate between them.

Reply:
ok